_posts/2023-11-27-Announcing-Data-Prepper-2.6.0.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.6.0"
+authors:
+- dlv
+date: 2023-11-28 13:30:00 -0600
+categories:
+  - releases
+meta_keywords: Data Prepper DynamoDB source, Kafka buffer, Amazon OpenSearch Serverless network policy
+meta_description: The Data Prepper maintainers launched Data Prepper 2.6.0 with DynamoDB ingestion, improved durability with a Kafka buffer, better support for Amazon OpenSearch Serverless, and more.
```

Comment:
Please update the meta with the following: 

**meta_keywords:** Data Prepper DynamoDB source, Apache Kafka buffer, OpenSearch Serverless network policy

**meta_description:** With Data Prepper 2.6.0 you can ingest data from DynamoDB, improve data durability using the new Apache Kafka buffer, and automatically connect to Amazon OpenSearch Serverless collections.

Reply:
Thank you @pajuric for the recommendations. I applied them in the latest commit.